docs/wiki/_Sidebar.md

```diff
@@ -25,6 +25,7 @@
   - [Expand Built-in Archetype Definitions][wiki_expand_built_in_archetype_definitions]
   - [Deploy Using Module Nesting][wiki_deploy_using_module_nesting]
   - [Override Module Role Assignments][wiki_override_module_role_assignments]
+  - [Multi-Subscription Deployment][wiki_multi-subscription_deployment]
```

Comment:
We should probably look at how to integrate this with the [Provider Configuration](https://github.com/krowlandson/terraform-azurerm-caf-enterprise-scale/wiki/%5BUser-Guide%5D-Provider-Configuration) page